### ISSUE TITLE
[Mechanics]Missing dialog option for ICC buff

### ISSUE THREAD

**Issue by Chenocide** [dalaranwow/dalaran-wow#4893]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Hellscream(/maybe Wrynn too) does not offer the option to cancel the ICC buff. 

**Expected behaviour**: Tell us what should happen instead

The buff should by default be turned on, then you can turn it off by talking to the respective faction NPCs.  

**Steps to reproduce the problem**:

1. Enter ICC.
2. Visually confirm the buff is active.
3. Talk to Hellscream/Wrynn. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

Hellscream ID: 39372
Wrynn ID: 39371

Comments from one of the ICC buffs:
http://www.wowhead.com/spell=73816/hellscreams-warsong#comments

Similar comments from another forum:
https://www.gamefaqs.com/boards/534914-world-of-warcraft/53806618

Alliance version of the missing dialog:
https://www.blogcdn.com/wow.joystiq.com/media/2010/03/no-wrynn-1267824916.jpg

**Include Screenshots from the issue if necessary**

https://www.youtube.com/watch?v=4VsARxj72EQ&feature=youtu.be

**Comment by Lothloryen** [comment]
Hello there,

It will be deployed during the next few days. Thank you for your report.

**Comment by Crispybuilt** [comment]
Deployed to Live.
